### DISCUSSION TITLE
Is it possible to use FirstPersonView to look at a point?

### DISCUSSION THREAD

**Discussion by chrisgervang** [visgl/deck.gl#5293] [Q&A]
I currently use the FirstPersonView to position the camera with geospatial coords and have it look towards a direction and orientation, as intended. I was wondering if there was a function to compute the pitch and bearing in the direction of at a particular lat/lng on the ground nearby?

**Comment by ibgreen** [comment]
Check out https://math.gl/modules/web-mercator/docs/api-reference/web-mercator-utils#getdistancescalesviewport

Once you have distance scales you can convert the long/lat deltas to meter deltas along X and Y axis. Assuming your elevation is in meters along Z axis you now just have to do the simple trigonometry to find out what you need.
